pkg/app/piped/notifier/slack.go

```diff
@@ -51,14 +53,30 @@ type slack struct {
 }
 
 func newSlackSender(name string, cfg config.NotificationReceiverSlack, webURL string, logger *zap.Logger) *slack {
+	var oauthtoken string
+	if cfg.OAuthTokenData != "" {
+		a, err := base64.StdEncoding.DecodeString(cfg.OAuthTokenData)
```

Comment:
The ident `a` is unclear. Please declare clear ident.

Reply:
thanks for your feedback, I declared a new ident